ros_msft_onnx/cfg/reconfig.cfg

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env python
+PACKAGE = "ros_msft_onnx"
+ 
+from dynamic_reconfigure.parameter_generator_catkin import *
+import os
+
+os.chdir('../../../')
+path = os.getcwd()
+
+newPath = path.replace(os.sep, '/') + "/src/ros_msft_onnx/ros_msft_onnx/testdata/"
+gen = ParameterGenerator()
+ 
+gen.add("tensor_height",    int_t,    0, "The height of the tensor", 416, 0, 3072)
+gen.add("tensor_width",    int_t,    0, "The width of the tensor", 416, 0, 4096)
+gen.add("confidence", double_t, 0, "The confidence parameter",    .5, 0, 1)
+gen.add("onnx_model_path",    str_t,    0, "The path of the onnx model",  newPath + "model.onnx")
+gen.add("onnx_label_path",    str_t,    0, "The path of the class labels",  newPath + "labels.txt")
+gen.add("tracker_type",    str_t,    0, "Name of the tracker",  "yolo")
+gen.add("image_processing",    str_t,    0, "Changes the image processing type",  "resize")
+gen.add("input_node_name",    str_t,    0, "Name of the input for onnx model",  "image")
+gen.add("output_node_name",    str_t,    0, "Name of the output for onnx model",  "grid")
+gen.add("debug",   bool_t,   0, "Sets the debug value",  True)
+gen.add("anch0", double_t, 0, "anchor0", 1.08, 0, 20)
```

Comment:
Can we move the anchors into a config file?

Reply:
@ooeygui moved the anchors to a yaml file which addresses the below comment as well?